src/command_modules/azure-cli-appservice/azure/cli/command_modules/appservice/_params.py

```diff
@@ -103,12 +103,19 @@ def get_hostname_completion_list(prefix, action, parsed_args, **kwargs):  # pyli
     register_cli_argument(scope + ' deployment source', 'manual_integration', action='store_true', help='disable automatic sync between source control and web')
     register_cli_argument(scope + ' deployment source', 'repo_url', options_list=('--repo-url', '-u'), help='repository url to pull the latest source from, e.g. https://github.com/foo/foo-web')
     register_cli_argument(scope + ' deployment source', 'branch', help='the branch name of the repository')
-    register_cli_argument(scope + ' deployment source', 'cd_provider', help='type of CI/CD provider', default='kudu', **enum_choice_list(['kudu', 'vsts']))
-    register_cli_argument(scope + ' deployment source', 'cd_app_type', arg_group='VSTS CD Provider', help='web application framework you used to develop your app', default='AspNetWap', **enum_choice_list(['AspNetWap', 'AspNetCore', 'NodeJSWithGulp', 'NodeJSWithGrunt']))
-    register_cli_argument(scope + ' deployment source', 'cd_account', arg_group='VSTS CD Provider', help='name of the Team Services account to create/use for continuous delivery')
-    register_cli_argument(scope + ' deployment source', 'cd_account_must_exist', arg_group='VSTS CD Provider', help='specifies that the account must already exist. If not specified, the account will be created if it does not already exist (existing accounts are updated)', action='store_true')
-    register_cli_argument(scope + ' deployment source', 'repository_type', help='repository type', default='git', **enum_choice_list(['git', 'mercurial']))
-    register_cli_argument(scope + ' deployment source', 'git_token', help='git access token required for auto sync')
+    register_cli_argument(scope + ' deployment source', 'private_repo_username', arg_group='VSTS CD Provider', help='Username for the private repository')
+    register_cli_argument(scope + ' deployment source', 'private_repo_password', arg_group='VSTS CD Provider', help='Password for private repository. Will prompt if not provided')
```

Comment:
For `Will prompt if not provided`, I am not seeing any code here doing the prompt, I assume your SDK library will do it?